### PR TITLE
B1

### PR DESCRIPTION
**Pull request by sshort42**

### CODE REVIEW

File: file1 [hunk 1]
```diff
@@ -10,13 +10,13 @@ be reincarnated.
 4. Drop rod. Get bird. Get rod.
 5. Wave rod. Drop rod. Go get diamonds and gold.
 6. Release bird. Drop cage. Grab coins and silver. Note: Dwarves are lousy
-shots, but sometimes, unfortunately, they can get lucky.  Carry the ax, as you
```
Comment (sshort42) [inline file1:13]: Are you sure this is how ax (axe) is spelt (spelled) ?

File: file3 [hunk 2]
```diff
@@ -142,11 +142,9 @@ Put In Nickel To Operate
 PLYMOUTH
 --------
 Please Let Your Mother Out from Under The Hood
-Pussy Lips in Your MOUTH
 
 PONTIAC
 -------
-Poor Old Nigger Thinks It's A Cadillac
```
Comment (sshort42) [inline file3:149]: Good to get rid of this.

File: file1 [hunk 3]
```diff
@@ -10,13 +10,13 @@ be reincarnated.
...
 may encounter up to five of these critters.
-7. Try the magic word PLUGH also. Drop silver, gold nugget, diamonds, jewelry,
+7. Try the magic word PLUGH also. Drop silver, gold nugget, diamonds, jewellery,
 and coins. If you haven't already, you will soon encounter a thieving pirate.
 Not to worry, he's got to rob you at least once if you're to win all the
 marbles.
-8. Carrying food, ax, bottle of water, key (all for later) and lantern, visit
+8. Carrying food, axe, bottle of water, key (all for later) and lantern, visit
 the software den (Microsoft Version only). Don't mess with anything--Software
 types are a weird lot.  Get magic word LWPI.  Works only from here.
```
Comment (sshort42) [inline file1:21]: Add comment about what LWPI means?

**Review by sshort — CHANGES_REQUESTED** [review]
Please address the comments

File: file1 (hunk 1 above)
Comment (sshort42) [reply file1:13]: I'm happy with this

File: file1 (hunk 3 above)
Comment (sshort42) [reply file1:21]: It's a magic work - 'nuff said.

**Review by sshort — APPROVED** [review]
OK - looks good.

**Pull request merged**